libs/design/scss/theming/README.md

```diff
@@ -1,4 +1,4 @@
-# Theming
+# Daffodil Theming
 Daffodil's theming capabilities enables you to customize `@daffodil/design` components to reflect your brand. A theme consists of custom color configurations that will work in dark and light themes.
 
 ## Custom Colors
```

Comment:
This is broken.

Reply:
What do you mean broken?

Reply:
The link. GH didn't do a good job referencing the line:

:stop: Before you begin, read the [accessibility guide on color in @daffodil/design](https://github.com/graycoreio/daffodil/blob/43048f9ee6bc0753672aa8f8bb7674f8878a062e/libs/design/guides/color#accessibility.md).